servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -369,42 +387,61 @@ public <R> R getCommitLog(
       String pageToken,
       PagedResponseHandler<R, LogEntry> pagedResponseHandler)
       throws NessieNotFoundException {
-    // we should only allow named references when no paging is defined
-    WithHash<NamedRef> endRef =
-        namedRefWithHashOrThrow(namedRef, null == pageToken ? youngestHash : pageToken);
+    // FIXME we should only allow named references when no paging is defined
```

Comment:
TBH; I'm not sure that comment is still valid (aka: I think, it can just go away)

Reply:
I thought about that too and came to the conclusion that it could still be relevant. If the named reference is reassigned between two pages to a completely different hash, paging could break I guess. Wdyt?

Reply:
Paging for commit-log uses a commit-hash passed around as the offset for the next page. It could be an issue when the reference is deleted, but then it's deleted, so fine to fail IMO.

Reply:
OK let's remove the comment then.